src/Workspaces/Core/Desktop/Workspace/Host/TemporaryStorage/TemporaryStorageServiceFactory.MemoryMappedInfo.cs

```diff
@@ -99,13 +99,16 @@ public Stream CreateReadableStream()
                 // CreateViewAccessor is not guaranteed to be thread-safe
                 lock (_memoryMappedFile)
                 {
-                    if (_accessor == null)
+                    var streamAccessor = _accessor.TryAddReference();
```

Comment:
:arrow_right: The stream owns this reference, and it's the only thing that's done with it. This seems to be covered by the existing documentation comment for the method.